assets/src/components/p4_table/edit.js

```diff
@@ -35,19 +35,6 @@ import {
 	ToolbarGroup,
 	ToolbarItem,
 } from '@wordpress/components';
-import {
```

Comment:
Didn't this work? We're modifying it to pass strings, but as far as I can tell it's doing the same thing.